source/extensions/access_loggers/common/grpc_access_logger.h

```diff
@@ -172,16 +191,18 @@ class GrpcAccessLogger : public Detail::GrpcAccessLogger<HttpLogProto, TcpLogPro
                    std::chrono::milliseconds buffer_flush_interval_msec,
                    uint64_t max_buffer_size_bytes, Event::Dispatcher& dispatcher,
                    Stats::Scope& scope, std::string access_log_prefix,
-                   const Protobuf::MethodDescriptor& service_method)
+                   const Protobuf::MethodDescriptor& service_method,
+                   const absl::optional<envoy::config::core::v3::RetryPolicy> retry_policy)
```

Comment:
Here and in other spots:  several copies of retry_policy are being made -- perhaps pass it by reference or do a std::move?